src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -8472,6 +8473,87 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  '/nodes/{nodeId}/linked-rules':
```

Comment:
I'm not sure how does look the convention for REST APIs in Alfresco, but what about path: `/nodes/{nodeId}/rules/link`. It looks simple and quite intuitive.

Reply:
After the meeting I like the idea of: `/nodes/{nodeId}/rulesets/{rulesetId}/links`